elastic/kibana
# [Fleet] Use new shortcut links to Fleet discuss forums.

## Summary

Fixes https://github.com/elastic/kibana/issues/90731 part c).

This change updates the shortcut links used in the Fleet UI. They are now https://ela.st/fleet-forum and https://ela.st/fleet-feedback, both pointing to https://discuss.elastic.co/tags/c/elastic-stack/81/fleet

## How to test this

Find the links to the discuss forums and verify they lead to the correct URL.
* go to any integration detail page, click "See more details" at the very bottom to open a flyout. In the flyout, click "Discuss forums".
* on any Fleet page, click on the "Send feedback" link in the upper right

(Thanks @FrankHassanabad for pointing out the issue to us.)
